### PR TITLE
Add .codecov.yml file

### PR DESCRIPTION
**Pull request by jarifibrahim**
Fixes https://github.com/coala-analyzer/coala-bears/issues/347

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    status: 
+      change: 
+        enabled: false
+      patch: 
+        enabled: false
+      project: 
+        enabled: true
+        target: auto
+        threshold: 100%
+
```
Comment (sils1297) [inline .codecov.yml]: can you remove this line?

Comment (jarifibrahim) [inline .codecov.yml]: Done

**Comment by sils1297** [comment]
hey, thanks! Think we can merge it today.

**Comment by sils1297** [comment]
cancelling builds because of the other stuff going on

**Comment by jarifibrahim** [comment]
No problem.

**Comment by sils1297** [comment]
cherry picking into my next PR

**Pull request closed without merging**